code/datums/mood_events/generic_negative_events.dm

```diff
@@ -361,3 +361,12 @@
 	description = "<span class='boldwarning'>I hate being sprayed with water!</span>\n"
 	mood_change = -1
 	timeout = 30 SECONDS
+
+/datum/mood_event/gamer_withdrawal
+	description = span_warning("I wish I was gaming right now...\n")
+	mood_change = -5
+
+/datum/mood_event/gamer_lost
+	description = span_boldwarning("I hate losing after a heated gamer moment!\n")
```

Comment:
```suggestion
	description = span_boldwarning("If I'm not good at video games, can I truly call myself a gamer?\n")
```
you have the heated gamer moment after you lose, dummy

Reply:
the high-effort alternative here would be to make this description be randomly chosen from a list of moba (or artosis) salt rant transcripts